src/app/docs/key-info/key-info-docs.component.html

```diff
@@ -7,6 +7,44 @@
     The key info component highlights important summary information.
   </sky-docs-demo-page-summary>
 
+  <sky-docs-demo>
+    <sky-docs-demo-control-panel
+      (selectionChange)="onDemoSelectionChange($event)"
+    >
+      <sky-docs-demo-control-panel-section>
+        <sky-docs-demo-control-panel-radio-group
+          heading="Layout"
+          propertyName="layout"
+          [choices]="layoutChoices"
+          [initialValue]="'vertical'"
+        >
+        </sky-docs-demo-control-panel-radio-group>
+      </sky-docs-demo-control-panel-section>
+    </sky-docs-demo-control-panel>
+
+    <div
+      style="width:400px;"
+    >
+      <sky-key-info
+        [layout]="layout"
+      >
+        <sky-key-info-label>
```

Comment:
Are the selectors here correct? It seems like `575` should be the value and that `New members` should be the label. Same question for the code sample.

Reply:
Good catch! Fixed.